.env

```diff
@@ -25,6 +25,7 @@ DISABLE_ENTERPRISE_LOGIN=''
 ENABLE_COOKIE_POLICY_BANNER=''
 ENABLE_DYNAMIC_REGISTRATION_FIELDS=''
 ENABLE_PROGRESSIVE_PROFILING_ON_AUTHN=''
+ENABLE_PERSONALIZED_RECOMMENDATIONS=''
```

Comment:
This is missing from the config in https://github.com/openedx/frontend-app-authn/blob/master/src/config/index.js